doc/tutorial/concepts/index.md

```diff
@@ -246,7 +246,7 @@ if a numeric argument is an integral square (i.e. the user shall only be allowed
 
 ### Understanding the requirements
 
-In the previous section you analysed seqan3::validator.
+In the previous section, you analysed seqan3::validator.
```

Comment:
you sure about that comma? I'm no expert in grammar at all but it sounds obsolete

Reply:
The "correct" order would be `You analysed seqan3::validator in the previous section.`.
Because you change the ordering, you usually (not sure if you **have** to) put a comma there.